x/concentrated-liquidity/position_test.go

```diff
@@ -1192,6 +1212,239 @@ func (s *KeeperTestSuite) TestMintSharesLockAndUpdate() {
 	}
 }
 
+func (s *KeeperTestSuite) TestPositionHasActiveUnderlyingLock() {
```

Comment:
Is this overlapping with another PR? Skipping since I've already reviewed

Reply:
Yes, that was my bad. Sorry about that